http-basic-authentication/README.md

```diff
@@ -0,0 +1,68 @@
+# HTTP Basic Authentication
+
+[RFC 7617](https://datatracker.ietf.org/doc/html/rfc7617)
+
+[Golang Example](http-basic-authentication/main.go)
+
+> WARNING: HTTP Basic Auth is considered to be insecure. If you **must** use it, ensure you are using a TLS connection since it sends the password over the wire.
```

Comment:
```suggestion
> [!WARNING]
> HTTP Basic Auth is considered to be insecure. If you **must** use it, ensure you are using a TLS connection since it sends the password over the wire.
```